Erica content prep ideas (#5)
UWB-ACM/Hackathon-Docs-2020: _docs/hackathon-preperation-checklist.md

> # The Week Before

Reviewer: ```suggestion
# 1-2 Weeks Beforehand
```

> # The Week Before

Reviewer: Needs Jekyll headers here so that the page gets listed in the site menu.

````markdown
---
layout: default
category: Getting Started
title: Preparing for the Event
order: 3
permalink: /preparing.html
---
````

> - [x] testing task list

Reviewer: ```suggestion
```

- Make a team
- Brainstorm ideas

>     - [link to project-ideas page?]

Reviewer: ```suggestion
    - Not sure where to start? Check out our [project ideas page](_docs/project-ideas.md) to get your creative juices flowing.
```

> - Update your software

Reviewer: This is a bit vague; I'd nix it unless there are specific things that students should be updating or installing.
```suggestion
```

> - Get used to AWS?

Reviewer: Something along the lines of "sign up for an account with a cloud provider", and include the docs page which covers free account tiers and student discounts/accounts.
```suggestion
- Sign up for an account with a cloud provider
```

>     - something to do with getting used to cloud technology

Reviewer: ```suggestion
    - Check out our [provider overview](_docs/cloud_account_setup.md) for more information about free tier access and account credits
```

# Dawn of the Final Day

> blurb about stuff to do day before event

Reviewer: To do: 1-2 sentence description, or delete altogether

> ## 24 hours remain

Reviewer: ```suggestion
## 24 Hours Remain
```

> - stuff to do the day before

Reviewer: ```suggestion
```

- Charge devices
- Make a playlist
- Check-in with group

> ### What to Bring

Reviewer: ```suggestion
### What to Bring to the Event
```

- Laptop(s), Power Cables, USB Cables, Mouse, (Mechanical) Keyboard, HDMI Cable 

- Adapter Cables to full-size HDMI, USB
- A flash drive (for slides or quick file sharing)
- (Charged) Phone + Charger
- Headphones
- Water bottle
- Toiletries / Comfy clothes (still can be useful for a single-day event)

> Some hardware, like Raspberry Pis and the required components to use them will be provided to you. AWS will be provided to you. Food will be provided, but if you have some dietary restrictions, plan accordingly.

Reviewer: I fleshed out some of these comments. Note that we will not be providing AWS access (or any other service providers); students must set that up on their own. See changes under "1-2 Weeks Beforehand".
```suggestion
Some hardware, like Raspberry Pis and the required components to use them, will be provided to teams on a first-come, first-served basis if you wish to incorporate them in your project. The devices must be returned to the organizers at the end of the event.

Food will be provided, but if you have some dietary restrictions, plan accordingly. Contact the hackathon organizers for specific questions about dietary needs.
```

> The UWB Hacks is a 13-hour event, which can be a little daunting. Not sure how to prepare? Don't worry, this list has you covered.

Reviewer: Needs a Jekyll template header here for the file to be added to the navigation menu.

```suggestion
---
layout: default
category: Getting Started
title: Preparing to Hack
order: 3
permalink: /preparing.html
---

The UWB Hacks is a 13-hour event, which can be a little daunting. Not sure how to prepare? Don't worry, this list has you covered.
```

# 1-2 Weeks Before

- Make a team
    - 6 should be the max, but any group size below that is fine.
- Brainstorm ideas

>     - Not sure where to start? Check out our [project ideas page](_docs/project-ideas.md) to get your creative juices flowing.

Reviewer: Use the HTML permalink for internal linking; I figured out earlier this week that linking to the `.md` file will cause a 404 error, because all the markdown docs are converted by Jekyll to HTML pages for us, and the pages are generated according to the permalinks we assign them.
```suggestion
    - Not sure where to start? Check out our [project ideas page](project_ideas.html) to get your creative juices flowing.
```

- Get reacquainted with your IDE, and what programming language your team plans on using.
    - If it's been a while, try out a "Hello World", just to ensure your development environment works.
- Sign up for an account with a cloud provider

>     - Check out our [provider overview](_docs/cloud_account_setup.md) for more information about free tier access and account credits

Reviewer: ```suggestion
    - Check out our [provider overview](cloud_setup.html) for more information about free tier access and account credits
```

# Dawn of the Final Day
So you've got your teams picked out, and the Hackathon is in the morning. What should you do the day before to prepare?

## 24 Hours Remain
- Charge your devices
    - This includes: laptops, phones, battery packs, etc.
- Check-in with your group
- Make a playlist

### What to Bring to the Event
- Laptop(s)
- Power Cables
- USB Cables
- Mouse
- (Mechanical) Keyboard
- HDMI Cable 
    - Adapter Cables to full-size HDMI, USB
- A flash drive (for slides or quick file sharing)
- (Charged) Phone + Charger
- Headphones
- Water bottle
- Toiletries / Comfy clothes (still can be useful for a single-day event)

Some hardware, like Raspberry Pis and the required components to use them, will be provided to teams on a first-come, first-served basis if you wish to incorporate them in your project. The devices must be returned to the organizers at the end of the event.

Food will be provided, but if you have some dietary restrictions, plan accordingly. Contact the hackathon organizers for specific questions about dietary needs.

> 

Reviewer: ```suggestion
### Additional Resources
```
